### DISCUSSION TITLE
useAsyncValidator Schema is not a constructor

### DISCUSSION THREAD

**Discussion by compilekaiten** [vueuse/vueuse#2402] [Q&A]
After adding `useAsyncValidator` to my project, I am unable to build using Vite-SSG becuase of this error.
I'm not sure how to fix it.
```
[Vue warn]: Unhandled error during execution of watcher callback 
  at <ContactForm>
file:///C:/Users/redacted/Desktop/project/node_modules/.pnpm/@vueuse+integrations@9.4.0_vby4ubxt75z5eqbw75oifiymwa/node_modules/@vueuse/integrations/useAsyncValidator.mjs:40
    const validator = new Schema(resolveUnref(rules));
                      ^

TypeError: Schema is not a constructor
```

**Comment by joshistoast** [comment]
I'm also stuck on this, just started trying it out today 😢

**Comment by theruziev** [comment]
https://github.com/yiminghe/async-validator/issues/312 - found an issue in async-validator repository. 
I used it in my nuxt3 project and only disabled ssr fixed this. maybe it depends to ssr

**Reply by theruziev** [reply]
also created a bug https://github.com/vueuse/vueuse/issues/2497
